packages/nocodb-sdk/src/lib/globals.ts

```diff
@@ -13,6 +13,7 @@ export enum RelationTypes {
 
 export enum ExportTypes {
   CSV = 'csv',
+  CSV_DATA = 'csvData',
```

Comment:
Should it be `EXCEL`?

Reply:
As I said, the backend doesn't build the excel file. But maybe it'd be better to do so ?